data/SIMS Activation Log.csv

```diff
@@ -0,0 +1,34 @@
+Year,Country,Code,Response,Support,Significance (1-5; 5 as most significant activations),Start Date,End Date
```

Comment:
Yes I originally wanted to do that but wasn't sure how to do it. Happy to learn how to do it from you! The person maintaining the sheet and the severities is @pknight03, he can provide you with the original document :)